website/src/preferences/code-hosting-origin-hostname.md

```diff
@@ -1,10 +1,14 @@
 # code-hosting-origin-hostname
 
 ```
-git-town.code-hosting-origin-hostname=<name>
+git-town.code-hosting-origin-hostname=<hostname>
 ```
 
-When using SSH identities, you can override the hostname of your source code
-repository using this configuration setting. To define it, run
-`git config git-town.code-hosting-origin-hostname <hostname>` where hostname
-matches what is in your ssh config file.
+When using SSH identities, you can use this configuration setting to define the
+hostname of your source code repository by running:
+
+```
+git config [--global] git-town.code-hosting-origin-hostname <hostname>
+```
+
+`<hostname>` should match the hostname in your ssh config file.
```

Comment:
Should we also have the explanation of the optional global flag here?

Reply:
Good catch, resolved in https://github.com/git-town/git-town/pull/1870